src/native/corehost/fxr/fx_resolver.cpp

```diff
@@ -286,7 +287,6 @@ namespace
 
         if (selected_fx_dir.empty())
         {
-            trace::error(_X("It was not possible to find any compatible framework version"));
```

Comment:
I moved the user-facing error to where we were displaying the details instead - line 446. I can bring this back and turn it into a verbose trace instead, so that this function always logs some verbose message before returning its result.

Reply:
Entirely up to you all. Just curious.